Documentation/Signoffs/Motor System.md

```diff
@@ -58,7 +97,7 @@ This is the wiring diagram for the driver and how it connects to the motor and s
 
 ## Analysis
 
-As stated previously, the motor must be able to output enough torque to rotate roughly 45 lbs. Our current machine design is a hollow cylinder, with the outer diameter being 3' and the inner, hollow diameter being roughly 1'4"  
+As stated previously, the motor must be able to output enough torque to rotate roughly 75 lbs. Our current machine design is a hollow cylinder, with the outer diameter being 3' and the inner, hollow diameter being roughly 1'4"  
 
 T=W(R1^2+R2^2)/2∗∆N/308t
```

Comment:
Make this an equation in the same way mentioned above. Also, this seems to be pulled from nowhere. What is this equation?